docs/milestones.md

```diff
@@ -0,0 +1,23 @@
+# Milestones
+
+- [ ] [[M0] Diseño de las estructuras de datos en ficheros](https://github.com/danieeeld2/LogisticsRoutes/milestone/1)
+  
+  _Teniendo en cuenta lo hablado con los usuarios y el contexto que estos nos proporcionan, necesitamos las siguientes estructuras de datos:_ 
+  
+  - Una estructura de datos que nos permita obtener el recorrido más eficiente entre el punto de inicio y el punto de entrega (_Modeliza las carreteras_)
+  
+  - Una estructura que almacene las especificaciones del vehículo (_dimensión, peso, indicación de que transporta mercancía peligrosa o no_)
```

Comment:
Que las especificaciones del vehículo estén en una o en tres estructuras de datos, o no estén, tiene que decidirse a la hora de modelizar. No especifiquéis absolutamente nada más que el resultado.